### PR TITLE
V3: feat: allow range input no filter

### PR DESCRIPTION
**Pull request by tuanddd**
## Should merge #194 first.

### CODE REVIEW

File: docs/pages/components/rangeinput.mdx [hunk 1]
```diff
@@ -29,10 +29,77 @@ function Example() {
...
+### Usage without Filter controller
+
+In case you don't want to depend on `RangeFilter`, you can otp-out of that by passing `limit`, `value`, `onChange` to the `RangeInput`.
+
+```jsx
+function Example() {
+  const [range, setRange] = React.useState([200, 100]);
+  return (
+    <div className="flex flex-col">
+      <RangeInput limit={[0, 500]} value={range} onChange={setRange} />
```
Comment (sampotts) [inline docs/pages/components/rangeinput.mdx:87]: Should `limit` really be split into `min` and `max` instead so we're similar to the native element's API?

Comment (tuanddd) [reply docs/pages/components/rangeinput.mdx:87]: Isn't the `limit` props already include the `min` and `max`? 🤔
The syntax is `[min, max]` but if you wanna split that up that's fine also fine 🙂

File: docs/pages/components/rangeinput.mdx [hunk 2]
```diff
@@ -29,10 +29,78 @@ function Example() {
...
+}
+```
+
+## Props
+
+| Name         | Type                                     | Default    | Description                                                                                       |
+| ------------ | ---------------------------------------- | ---------- | ------------------------------------------------------------------------------------------------- |
+| `min`        | `number`                                 | `0`        | The min range, used when you want to use the component without RangeFilter                        |
+| `max`        | `number`                                 | `100`      | The max range, used when you want to use the component without RangeFilter                        |
+| `value`      | `[number, number]`                       | `[25, 50]` | The current value of the range input, used when you want to use the component without RangeFilter |
```
Comment (sampotts) [inline docs/pages/components/rangeinput.mdx:102]: Sorry, I should have mentioned this earlier, is there a way to make this a single value if you just want one slider?

File: docs/pages/components/rangeinput.mdx [hunk 3]
```diff
@@ -29,10 +29,78 @@ function Example() {
...
+| `onChange`   | `(value: [number, number]) => void`      | `() => {}` | The callback when the value changes, used when you want to use the component without RangeFilter  |
```
Comment (sampotts) [inline docs/pages/components/rangeinput.mdx:103]: Same as above I guess for the value

File: packages/components/src/RangeInput/defaults.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { Range } from '@sajari/react-hooks';
+
+export const defaultParams = {
+  onChange: () => {},
+  value: [25, 50] as Range,
+  limit: [0, 100] as Range,
```
Comment (sampotts) [inline packages/components/src/RangeInput/defaults.ts:6]: I guess this needs updating?

**Pull request merged**